XIVSlothCombo/Combos/PLD.cs

```diff
@@ -246,9 +248,11 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                 if (IsEnabled(CustomComboPreset.PaladinHolyCircleFeature) && HasEffect(PLD.Buffs.Requiescat) && level >= PLD.Levels.HolyCircle && LocalPlayer.CurrentMp >= 1000)
                 {
-                    if (((IsEnabled(CustomComboPreset.PaladinAoEConfiteorFeature) && level >= PLD.Levels.Confiteor  &&
-                        GetBuffRemainingTime(PLD.Buffs.Requiescat) <= 3) || GetBuffStacks(PLD.Buffs.Requiescat) is 1 || LocalPlayer.CurrentMp <= 2000))
-                        return PLD.Confiteor;
+                    if ((IsEnabled(CustomComboPreset.PaladinAoEConfiteorFeature) && level >= PLD.Levels.Confiteor))
+                    {
+                        if (GetBuffRemainingTime(PLD.Buffs.Requiescat) <= 3 || GetBuffStacks(PLD.Buffs.Requiescat) is 1 || LocalPlayer.CurrentMp <= 2000)
+                            return PLD.Confiteor;
+                    }
```

Comment:
No need for the embedded `if` - you can resolve this with the following:
```if (((IsEnabled(CustomComboPreset.PaladinAoEConfiteorFeature) && level >= PLD.Levels.Confiteor  && (GetBuffRemainingTime(PLD.Buffs.Requiescat) <= 3) || GetBuffStacks(PLD.Buffs.Requiescat) is 1 || LocalPlayer.CurrentMp <= 2000)))```
```return PLD.Confiteor;```

formatting isn't ideal, but to be clear, the only change here is some parentheses wrapping the checks after the level check - `(` before `GetBuffRemainingTime` and `)` at the very end of the `if` statement.